src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="/">
```

Comment:
Use `#` instead, cause now if I click on logo or any nav link, there is a 404 error:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/157238908-a3aff7c5-2866-4986-a224-1e7beafc3659.png">


```suggestion
      <a class="header__logo" href="#">
```